nvOC_miner_update.sh

```diff
@@ -204,8 +313,16 @@ if [ ! $(cat ${NVOC_MINERS}/bminer/latest/version | grep 8.0.0) ]
 then
   echo "Extracting Bminer"
   mkdir -p ${NVOC_MINERS}/bminer/latest/
```

Comment:
If you intend to keep different bminer versions take a look at the ethminer symlink approach, otherwise using a "latest" folder is messy.

Reply:
Dont have time to make the changes now.
Pushing this PR and make the change in another PR.